cmd/carwings/main.go

```diff
@@ -64,13 +64,21 @@ func main() {
 		cfg                 config
 		username, password  string
 		region, sessionFile string
+		homeFolder          string
 	)
 
+	homeFolder, err := os.UserHomeDir()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "ERROR: could not get user homedir\n")
+		os.Exit(1)
+	}
+	fmt.Fprintf(os.Stderr, "User homedir found: "+homeFolder+"\n")
```

Comment:
Do you mind removing this line?  I think it's going to be too verbose for normal operation.